lib/src/generated/top_level.dart

```diff
@@ -4,13 +4,13 @@
 
 // Don't modify this file by hand! It's generated by tool/generate.dart.
 
-import '../../term_glyph.dart' as glyph;
+import '../../term_glyph.dart';
```

Comment:
This is a generated file but these look like hand edits? I don't see anything that looks like it would have changed the output from `tools/generate.dart`.

Reply:
oh this is the generated file? Ugh. Should have read the comments 😁

Reply:
PTAL